thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/performanceEvaluation/MergedAnomalyIntervalUtils.java

```diff
@@ -0,0 +1,98 @@
+package com.linkedin.thirdeye.anomalydetection.performanceEvaluation;
+
+import com.linkedin.thirdeye.api.DimensionMap;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Stack;
+import org.joda.time.Interval;
+
+
+public class MergedAnomalyIntervalUtils {
```

Comment:
Please consider Restrict this class to IntervalUitls, that extend the method that can be applied to Intervals. This method to merge intervals (or ranges) have a broader use in thirdeye, anomaly-detection code base and should abstracted out.